Clicker/Views/Cards/SeparatorLineCell.swift

```diff
@@ -29,9 +28,8 @@ class SeparatorLineCell: UICollectionViewCell {
     }
     
     override func updateConstraints() {
-        let lineViewLeadingOffset = model.state == .settings ? lineViewSettingsInset : 0
         lineView.snp.makeConstraints { make in
-            make.leading.equalToSuperview().offset(lineViewLeadingOffset)
+            make.leading.equalToSuperview()
             make.bottom.trailing.equalToSuperview()
```

Comment:
can you chain the leading/bottom/trailing together?
`make.bottom.leading.trailing.equalToSuperview()`

Reply:
Addressed in most recent pull request!